templates/server-ingress.yaml

```diff
@@ -41,6 +41,12 @@ spec:
     - host: {{ .host | quote }}
       http:
         paths:
+        {{- range .extraBackends }}
```

Comment:
It's not obvious to me how this is different than the existing configurable `.paths`. Wondering if you can clarify?

Reply:
Sure. I think @davorceman described the problem this solves well in #361.

To clarify the difference: The problem with the existing configurable `.paths` is it's assumptions of the values made by '$servicename' and '$serviceport'.